src/meta/src/stream/fragmenter.rs

```diff
@@ -52,6 +52,16 @@ pub struct StreamFragmenter<S> {
     /// local actor id
     next_local_actor_id: u32,
 
+    /// Next local table id to be allocated. It equals to total table ids cnt when finish stream
+    /// node traversing.
+    next_local_table_id: u32,
```

Comment:
`next_local_table_id` seems a little bit confusing. 🤔

Reply:
Rename to next table id